### PR TITLE
Automation cy sc

### PR DESCRIPTION
**Pull request by swathichelika**
**Note**: "file already attached" message isn't appearing even after the same file is attached again.

## Summary

Added automation script for Compose Reply with Attachment error validation

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#0000
- *Link to ticket created in va.gov-team repo*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*

## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

| | Before | After |
| --- | --- | --- |
| Mobile | | |
| Desktop | | |

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      .first()
+      .click();
+    cy.get('[data-testid="attach-file-input"]').selectFile(
+      'src/applications/mhv/secure-messaging/tests/e2e/fixtures/test_image_10mb.jpg',
+      { force: true },
+    );
+    cy.get('[data-testid="attach-file-error-modal"] p')
+      .should(
+        'have.text',
+        'File size for a single attachment cannot exceed 6MB',
```
Comment (Asterix22) [inline src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js:42]: Can you refactor the error message checks to be encapsulated in a method--something like PatientComposePage.verifyAttachmentErrorMessage(<<expectedAttachmentErrorMessage>>)

cy.get('[data-testid="attach-file-error-modal"] p')
      .should(
        'have.text',
        'File size for a single attachment cannot exceed 6MB',
      )
      .should('be.visible');

File: src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        'have.text',
+        'File supported: doc, docx, gif, jpg, jpeg, pdf, png, rtf, txt, xls, xlsx',
+      )
+      .should('be.visible');
+    cy.get('[data-testid="attach-file-error-modal"]')
+      .shadow()
+      .find('[type="button"]')
+      .first()
+      .click();
+    cy.get('[data-testid="attach-file-input"]').selectFile(
```
Comment (Asterix22) [inline src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js:35]: Can you encapsulate the code for attaching a message into a method :  PatientComposeMessage.attachMessageFromFile(<<filePath>>)

**Review by Asterix22 — CHANGES_REQUESTED** [review]

File: src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js (hunk 1 above)
Comment (swathichelika) [reply src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js]: I have refactored the error message checks to encapsulate in a method.

File: src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js (hunk 2 above)
Comment (swathichelika) [reply src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js]: yes, encapsulated the code for attaching a message into a method.

**Review by Asterix22 — APPROVED** [review]
Looks great now!

**Pull request closed without merging**